spark/src/test/scala/org/apache/comet/objectstore/NativeConfigSuite.scala

```diff
@@ -119,15 +119,18 @@ class NativeConfigSuite extends AnyFunSuite with Matchers with BeforeAndAfterEac
 
   test("validity cache") {
     val hadoopConf = new Configuration()
-    hadoopConf.set("fs.s3a.endpoint", "https://acme.storage.com")
```

Comment:
This used to trigger a validate error but that changed in https://github.com/apache/datafusion-comet/pull/2288